src/EnergyPlus/DataSurfaces.hh

```diff
@@ -125,6 +170,37 @@ namespace DataSurfaces {
         Num // The counter representing the total number of surface class, always stays at the bottom
     };
 
+    enum class SurfaceFilter
```

Comment:
I guess I didn't really add this enumeration in this PR, I just used it more extensively.